lib/k8s-test-framework/src/pod.rs

```diff
@@ -54,6 +54,24 @@ where
     Ok(node.to_string())
 }
 
+/// Set label on all nodes to discover this label from the pod.
```

Comment:
This _technically_ should go into a new `node.rs` to be super correct, but I'd also not want to add another module for a single function - I feel like this framework is a bit complex as-is.